pilot/cmd/pilot-agent/main.go

```diff
@@ -92,6 +99,9 @@ var (
 	podNamespaceVar      = env.RegisterStringVar("POD_NAMESPACE", "", "")
 	istioNamespaceVar    = env.RegisterStringVar("ISTIO_NAMESPACE", "", "")
 	kubeAppProberNameVar = env.RegisterStringVar(status.KubeAppProberEnvName, "", "")
+	sdsEnabledVar        = env.RegisterBoolVar("SDS_ENABLED", false, "")
+
+	sdsUdsWaitTimeout = time.Minute
```

Comment:
No wait time for UDS - it'll not magically show up like cert.

Reply:
updated only wait for controlplane sidecar(when controlplane security is enabled)

Reply:
Sounds good - that's only used in the old installer, will not affect the new one ( code is already deprecated)